packages/react-query/src/useBaseQuery.ts

```diff
@@ -67,6 +67,13 @@ export function useBaseQuery<
 
   useClearResetErrorBoundary(errorResetBoundary)
 
+  if (
+    !client.getQueryState(options.queryKey) &&
+    defaultedOptions.enabled !== false
+  ) {
+    client.prefetchQuery(defaultedOptions)
```

Comment:
(I suspect this is what causes the unhandled promise rejection errors, but I am on my phone)